2529.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var maximumCount = function (nums) {
+  let negativeInt = 0;
+  let positiveInt = 0;
+
+  for (let i = 0; i < nums.length; i++) {
+    if (nums[i] < 0) {
+      negativeInt++;
+    } else if (nums[i] > 0) {
+      positiveInt++;
+    }
+  }
```

Comment:
You can also reduce the amount of this code by using `Array.filter()` 👍 
```suggestion
  const NegativeNums = nums.filter(num => num < 0);
  const PositiveNums = nums.filter(num => num > 0);
```